Formula/crystal-lang.rb

```diff
@@ -6,25 +6,8 @@ class CrystalLang < Formula
   head "https://github.com/crystal-lang/crystal.git"
 
   stable do
```

Comment:
@ilovezfs Thank you for the explanation. I created a separate PR here: https://github.com/Homebrew/homebrew-core/pull/4449